attestation/src/lib.rs

```diff
@@ -39,6 +39,9 @@ pub enum Error {
     /// Invalid DER payload
     InvalidDer(der::Error),
 
+    /// Invalid X.509 extension DER
```

Comment:
Not related, but pedantic nit about commit message: "The API is minimal for now, it" should be "The API is minimal for now, **and** it"